K/Kronecker/Compat.toml

```diff
@@ -1,2 +1,5 @@
 [0]
 julia = "1"
+
+["0.4-0"]
+Documenter = "0.23"
```

Comment:
You probably don't want Documenter as a dependency, right?

Reply:
Added this in experimentation to generate the docs, should indeed be removed (done in the latest commit).

Reply:
Also got the following warning:
```
Your new version pull request does not meet the following guidelines for auto-merging:

The following dependencies do not have a `compat` entry that has an upper bound: FillArrays, StatsBase
```
How can I fix this or is this not critical?

Reply:
Its not critical, but its recommended and your PR will be merged automatically.